index.html

```diff
@@ -201,8 +202,43 @@ <h4 class="connect">LET'S CONNECT</h4>
               Skills <img src="./images/arrow-forward.png" alt="" />
             </li>
           </ul>
+          </div>
         </section>
       </section>
+
+      <section class="contact">
+        <h1 class="title-contact">Contact me</h1>
+        <p class="contact-info">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/mdojkaer" method="POST" class="contact-form">
+          <input
```

Comment:
- [ ] Kindly add the `required` attribute in your inputs to active validation on submit.